EpiResponsivePicture/ResizedImage/AspectRatio.cs

```diff
@@ -12,9 +14,9 @@ private AspectRatio(double ratio)
         Ratio = ratio;
     }
 
-    public static AspectRatio Original => new(-1);
+    public static AspectRatio Original() => new(-1);
 
-    public static AspectRatio Default => Original;
+    public static AspectRatio Default() => Original();
```

Comment:
I assume that conversion from property to a method is because of how these two behave when exporting to TS by ReinforcedTypings and we don't want to have `Original` and `Default` in TS?
The issue here is that this is kind of breaking change - everyone that has been using nuget so far will be forced to convert `Original` to `Original()`. Can we keep these two as properties and mark them with [TsIgnore] if we don't want it in TS?

Reply:
yes, exactly. using TsIgnore is much clearer solution. Done